examples/keras_simple.py

```diff
@@ -0,0 +1,81 @@
+"""
+Optuna example that optimizes a neural network classifier configuration for the
+MNIST dataset using Keras.
+
+In this example, we optimize the validation accuracy of MNIST classification using
+Keras. We optimize the filter and kernel size, kernel stride and layer activation.
+
+We have following two ways to execute this example:
+
+(1) Execute this code directly.
+    $ python keras_simple.py
+
+
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize keras_simple.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+
+"""
+
+import numpy as np
+
+from sklearn.metrics import accuracy_score
+
+from keras.datasets import mnist
+from keras.layers import Conv2D
+from keras.layers import Dense
+from keras.layers import Flatten
+from keras.models import Sequential
+from keras.optimizers import Adam
+
+import optuna
+
+
+def objective(trial):
```

Comment:
Please add the following line to clean the session graphs. I have encountered out-of-memory errors when I optimized keras models without this workaround.

```python
    # Clear clutter form previous session graphs.
    keras.backend.clear_session()
```